apps/fabric-website/src/pages/Controls/AvatarPage/docs/android/AvatarOverview.md

```diff
@@ -2,4 +2,4 @@
 
 | Avatar style examples                                                                                                                                                                                            |                                                                                                                                                                                                                  |
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/persona/avatar-circle.png" alt="Example of Circle Avatars in varying sizes on Android" style="width: 50%;" /> | <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/persona/avatar-square.png" alt="Example of Square Avatars in varying sizes on Android" style="width: 50%;" /> |
+| <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/persona/avatar-circle.png" alt="Example of Circle Avatars in varying sizes on Android" style="width: 75%;" /> | <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/persona/avatar-square.png" alt="Example of Square Avatars in varying sizes on Android" style="width: 75%;" /> |
```

Comment:
I think it's because they are in table cells, which puts them side by side.
The issue was that these screenshots were too small at 50% in tables. Bumping them to 75% make them more legible.